commit 4ad2b82f02940919076bbd75002408a5bfef6a80 (refs/changes/69/3569/2)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-03-09T04:38:49+00:00 (1 year, 7 months ago)
    
    OS-6747 make clean removes brand hooks
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/lib/brand/bhyve/zone/Makefile b/usr/src/lib/brand/bhyve/zone/Makefile
index f778fda27a..d7e957072c 100644
--- a/usr/src/lib/brand/bhyve/zone/Makefile
+++ b/usr/src/lib/brand/bhyve/zone/Makefile
@@ -20,7 +20,7 @@ PROGS =		attach detach statechange uninstall
 PROG1 =		boot
 PROG2 =		bhhwcompat
 PROGS +=	$(PROG1) $(PROG2)
-CLEANFILES +=	$(PROGS)
+CLEANFILES +=	$(PROG1) $(PROG2)
 
 $(PROG1) :=	LDLIBS += -lnvpair
 $(PROG2) :=	CPPFLAGS = -I$(COMPAT)/freebsd -I$(CONTRIB)/freebsd \
